src/backend/InvenTree/InvenTree/helpers.py

```diff
@@ -176,6 +176,8 @@ def constructPathString(path: list[str], max_chars: int = 250) -> str:
 
 def getMediaUrl(filename):
     """Return the qualified access path for the given file, under the media directory."""
+    if settings.STORAGE_TARGET == 's3':
```

Comment:
I suggest that we add an enum for the valid `STORAGE_TARGET` values - so that we are not performing string comparisons against raw values in multiple places across the codebase.